src/accessibility/outputs.js

```diff
@@ -86,59 +95,68 @@ p5.prototype.textOutput = function(display) {
  *
- * <code class="language-javascript">gridOutput()</code> and <code class="language-javascript">gridOutput(FALLBACK)</code>
- * make the output available in <a href="https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API/Tutorial/Hit_regions_and_accessibility" target="_blank">
- * a sub DOM inside the canvas element</a> which is accessible to screen readers.
- * <code class="language-javascript">gridOutput(LABEL)</code> creates an
- * additional div with the output adjacent to the canvas, this is useful
- * for non-screen reader users that might want to display the output outside
- * of the canvas' sub DOM as they code. However, using LABEL will create
- * unnecessary redundancy for screen reader users. We recommend using LABEL
- * only as part of the development process of a sketch and removing it before
- * publishing or sharing with screen reader users.
+ * The general description includes the canvas size, canvas color, and number of
+ * shapes. For example,
+ * `gray canvas, 100 by 100 pixels, contains 2 shapes:  1 circle 1 square`.
+ *
+ * `gridOutput()` uses its table of shapes as a grid. Each shape in the grid
+ * is placed in a cell whose row and column correspond to the shape's location
+ * on the canvas. The grid cells describe the color and type of shape at that
+ * location. For example, `red circle`. These descriptions can be selected
+ * individually to get more details. This is different from
+ * <a href="#/p5/textOutput">textOutput()</a>, which uses its table as a list.
+ *
+ * A list of shapes follows the table. The list describes the color, type,
+ * location, and area of each shape. For example,
+ * `red circle, location = middle, area = 3 %`.
+ *
+ * The `display` parameter is optional. It determines how the description is
+ * displayed. If `LABEL` is passed, as in `gridOutput(LABEL)`, the description
+ * will be visible in a div element next to the canvas. Using `LABEL` creates
+ * unhelpful duplicates for screen readers. Only use `LABEL` during
+ * development. If `FALLBACK` is passed, as in `gridOutput(FALLBACK)`, the
+ * description will only be visible to screen readers. This is the default
+ * mode.
+ *
+ * Read
+ * <a href="/learn/labeling-canvases.html">How to label your p5.js code</a> to
+ * learn more about making sketches accessible.
  *
  * @method gridOutput
- * @param  {Constant} [display] either FALLBACK or LABEL
+ * @param  {Constant} [display] either FALLBACK or LABEL.
  *
  * @example
  * <div>
  * <code>
- * gridOutput();
- * background(148, 196, 0);
- * fill(255, 0, 0);
- * ellipse(20, 20, 20, 20);
- * fill(0, 0, 255);
- * rect(50, 50, 50, 50);
+ * function setup() {
+ *   gridOutput();
+ *   background(200);
+ *   fill(255, 0, 0);
+ *   circle(20, 20, 20);
+ *   fill(0, 0, 255);
+ *   square(50, 50, 50);
+ *
+ *   describe('A red circle and a blue square on a gray background.');
+ * }
  * </code>
  * </div>
  *
- *
  * <div>
  * <code>
- * let x = 0;
  * function draw() {
  *   gridOutput();
```

Comment:
Would it make sense to show the gridOutput(LABEL) usecase in one of the example snippets?